test/functional/wallet_taproot.py

```diff
@@ -183,10 +182,7 @@ def multi_a(k, hex_keys, sort=False):
 
 def compute_taproot_address(pubkey, scripts):
     """Compute the address for a taproot output with given inner key and scripts."""
```

Comment:
Could be done in a follow up PR (happy to do it) but the terminology we are using is "internal key" rather than "inner key". (There is also a `inner_keys` variable name to be replaced in [feature_taproot.py](https://github.com/bitcoin/bitcoin/blob/master/test/functional/feature_taproot.py#L1502).)